src/_posts/2019-06-12-progress-structure.md

```diff
@@ -0,0 +1,52 @@
+
+A detail that I believe is important is that your career advancement will not depend on someone of higher seniority level vacating a position. Each person progress is its own, and will not be affected by others progression.
+
+Originally I saw this as trying to get well rounded people that is useful on every possible scenario. But probably the broken comb metaphor is better suited on the long term. The less people you have in your company, the more important it is that anyone can do any job. But as you grow, that importance subsides.
+
+Finally, the competency matrix that we are using is not a tick-all-boxes idea. It provides a general idea of  expectations.
+
+##  It is a Marathon, not a sprint.
+I think this is something that needs to be understood. It takes years to reach Principal (the highest seniority) level. You need a broad knowledge and experience. You need to have seen multiple projects, with different characteristics, so for any new assignments you can reach into your own drawer of solutions, approaches, tricks to move forward satisfactorily a project. Trying to race here only leads to disappointment. Or as they saying goes: Rome wasn’t built in a day.
+
+## Mentorship
+Mentoring is one of the most important ways in which we can help people to improve and progress. Therefore, we are introducing mentoring (non-compulsory) for every level of seniority. A mentor will become a guide and helper, to allow the mentee to move forwards. It is important however that the mentee the one in charge of his/her own progression. The mentor is there to lend a hand, provide advice, and help with getting Codurance to provide the resource/opportunities/solutions that the mentee required.
+
+## Roll Out plan
+
+The roll out plan is all about getting everyone more or less aligned in terms of the future progression, and on the more appropriate seniority level. We do expect people to be all over the place right now, due to the fact that before each was left to find their own path. Furthermore, we have a competency matrix that is live, but doesn’t mean it is perfect. Not only do we need to train ourselves, but the model itself. So a 6 month period has been reserved to get everyone rolling with a plan of improvement, and get additional feedback in the process. An initial assignment of mentors (based on the mentees preferences) has been made, just to make it easy for everyone to start.
```

Comment:
"before each was" -> "before everyone was"

Reply:
"is live, but doesn’t mean it is" -> "is live, but that doesn’t mean it is"

Reply:
done